libs/common/utilities/src/css/mixin/title.mixin.css.ts

```diff
@@ -22,13 +22,13 @@ const titleSizes = {
         )}
     `,
     2: css`
-        font-size: 3.2rem;
+        font-size: var(--vl-title-h2-font-size, 3.2rem);
```

Comment:
Misschien werkt dit wel, maar dit is toch niet intiutief? 
- als we voor deze opzet gaan zou het ook wel doorgetrokken moeten worden naar de andere headings
- dit werkt enkel als je weet dat de variabele nergens gespecifieerd is en hij overal als default de fallback gebruikt (wat dus niet logisch is als de default de fallback is)

Het is mij zelfs een raadsel: in searchFilterStyles staat enerzijds --vl-title-h2-font-size: 1.6rem; maar anderzijds h2 met font-size: 3.2rem

Waarom kan je niet gewoon concreet doen wat je wil: op die plaats in de vl-search-filter de font-size de juiste grootte geven? Ik heb nog niet gevonden waar dat dan wel is?